NuGet.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="local" value="d:\localnuget" />
```

Comment:
temporary?

Reply:
Yes, all feed changes in reaction PRs are temporary and would be undone before the commit.